v5.0.0-beta.5 (#27938)
mui-org/material-ui: CHANGELOG.md

### [Versions](https://material-ui.com/versions/)

## 5.0.0-beta.5

<!-- generated comparing v5.0.0-beta.4..next -->

_Aug 24, 2021_

Big thanks to the 24 contributors who made this release possible. Here are some highlights ✨:

- 🐛 Fixed lots of regressions and bugs to get us closer to the [v5 stable release milestone](https://github.com/mui-org/material-ui/milestone/44)
- 🗎 Improved the docs and the migration guide for upgrading to v5

### `@material-ui/core@5.0.0-beta.5`

#### Breaking changes

> - <!-- 36 --> [core] Update `.browserslistrc` file (#27788) @DanailH

Author: Wasn't sure how to document this breaking change. Should we list all browsers, or?

Reviewer: I would include the list of supported platforms like https://next--material-ui.netlify.app/guides/migration-v4/#supported-browsers-and-node-versions does but compared to the previous 5.x beta.

Author: Updated.

Reviewer 2: To fix the rendering of the code part? 

```suggestion
- [core] Update `.browserslistrc` file (#27788) @DanailH
```

### [Versions](https://material-ui.com/versions/)

## 5.0.0-beta.5

<!-- generated comparing v5.0.0-beta.4..next -->

_Aug 24, 2021_

Big thanks to the 24 contributors who made this release possible. Here are some highlights ✨:

- 🐛 Fixed lots of regressions and bugs to get us closer to the [v5 stable release milestone](https://github.com/mui-org/material-ui/milestone/44)
- 📚 Improved the docs and the migration guide for upgrading to v5

### `@material-ui/core@5.0.0-beta.5`

#### Breaking changes

- <!-- 36 --> [core] Update `.browserslistrc` file (#27788) @DanailH

  The targets of the default bundle have changed:

  - Chrome 90 (up from 84)
  - Edge 91 (up from 85)
  - Safari 14 (macOS) (up from 13.1) and 12.4 (iOS) (up from 12.2)
  - Opera 76 (up from 70)

#### Changes

- <!-- 42 --> [AppBar] Fix transparency issue on dark mode (#27281) @will-amaral
- <!-- 29 --> Revert "[BottomNavigation] onClick does not fire if tapped while scrolling (#22524)" (#27690) @eps1lon
- <!-- 68 --> [Autocomplete] Add verbose warning for defaultValue (#27925) @vedadeepta

> - <!-- 43 --> [Autocomplete] Rename Value type to AutocompleteValue (#27804) @michaldudak

Reviewer 3: This is a breaking change

### [Versions](https://material-ui.com/versions/)

## 5.0.0-beta.5

<!-- generated comparing v5.0.0-beta.4..next -->

_Aug 24, 2021_

Big thanks to the 24 contributors who made this release possible. Here are some highlights ✨:

- 🐛 Fixed lots of regressions and bugs to get us closer to the [v5 stable release milestone](https://github.com/mui-org/material-ui/milestone/44)

> - 🗎 Improved the docs and the migration guide for upgrading to v5

Reviewer: The Emmoji is not supported by Ubuntu. 9% of users use Linux so we should make sure we use a smaller, more widely supported Emoji palette.

### `@material-ui/core@5.0.0-beta.5`

#### Breaking changes

- <!-- 36 --> [core] Update `.browserslistrc` file (#27788) @DanailH

The targets of the default bundle have changed.

It now supports the following minimum versions:

<!-- #stable-snapshot -->

- Chrome 90 (up from 84)
- Edge 91 (up from 85)
- Safari 14 (macOS) (up from 13.1) and 12.4 (iOS) (up from 12.2)
- Opera 76 (up from 70)
- and more (see [.browserslistrc (`stable` entry)](https://github.com/mui-org/material-ui/blob/HEAD/.browserslistrc#L11))

> If you need to support IE 11, check out our [legacy bundle](/guides/minimizing-bundle-size/#legacy-bundle).

Reviewer 2: The changelog is not rendered on our domain name.

```suggestion
If you need to support IE 11, check out our [legacy bundle](https://next.material-ui.com/guides/minimizing-bundle-size/#legacy-bundle).
```

> The targets of the default bundle have changed.
> 
> It now supports the following minimum versions:
> 
> <!-- #stable-snapshot -->
> 
> - Chrome 90 (up from 84)
> - Edge 91 (up from 85)
> - Safari 14 (macOS) (up from 13.1) and 12.4 (iOS) (up from 12.2)
> - Opera 76 (up from 70)
> - and more (see [.browserslistrc (`stable` entry)](https://github.com/mui-org/material-ui/blob/HEAD/.browserslistrc#L11))
> 
> If you need to support IE 11, check out our [legacy bundle](https://next.material-ui.com/guides/minimizing-bundle-size/#legacy-bundle).

Reviewer: ```suggestion
  The targets of the default bundle have changed:

  - Chrome 90 (up from 84)
  - Edge 91 (up from 85)
  - Safari 14 (macOS) (up from 13.1) and 12.4 (iOS) (up from 12.2)
  - Opera 76 (up from 70)
```

This focuses only on the change to the prior release and groups the description properly under the listitem

#### Changes

- <!-- 42 --> [AppBar] Fix transparency issue on dark mode (#27281) @will-amaral
- <!-- 29 --> Revert "[BottomNavigation] onClick does not fire if tapped while scrolling (#22524)" (#27690) @eps1lon
- <!-- 68 --> [Autocomplete] Add verbose warning for defaultValue (#27925) @vedadeepta
- <!-- 43 --> [Autocomplete] Rename Value type to AutocompleteValue (#27804) @michaldudak
- <!-- 41 --> [ButtonGroup] Allow `size` customization via module augmentation (#27834) @aaronlademann-wf
- <!-- 67 --> [InputBase] Preserve host state  when changing `rows` from undefined to defined (#27683) @eps1lon
- <!-- 18 --> [InputLabel] Apply `asterisk` class when `required` (#27738) @alexile
- <!-- 26 --> [Select] Fix NativeSelect propagating classes to the DOM element (#27797) @mnajdova
- <!-- 28 --> [Stack] Match the customization standard (#27777) @oliviertassinari
- <!-- 70 --> [SvgIcon] Apply custom color if defined in the theme (#27923) @eps1lon
- <!-- 65 --> [Switch] Add optional `track` slot to SwitchUnstyled (#27916) @michaldudak

> - <!-- 25 --> [Timeline] Fix color="inherit" on TimelineDot (#27794) @mnajdova

Reviewer 2: Lab?

Author: Oops yes

- <!-- 68 --> [Autocomplete] Add verbose warning for defaultValue (#27925) @vedadeepta
- <!-- 41 --> [ButtonGroup] Allow `size` customization via module augmentation (#27834) @aaronlademann-wf
- <!-- 67 --> [InputBase] Preserve host state  when changing `rows` from undefined to defined (#27683) @eps1lon
- <!-- 18 --> [InputLabel] Apply `asterisk` class when `required` (#27738) @alexile
- <!-- 26 --> [Select] Fix NativeSelect propagating classes to the DOM element (#27797) @mnajdova
- <!-- 28 --> [Stack] Match the customization standard (#27777) @oliviertassinari
- <!-- 70 --> [SvgIcon] Apply custom color if defined in the theme (#27923) @eps1lon
- <!-- 65 --> [Switch] Add optional `track` slot to SwitchUnstyled (#27916) @michaldudak
- <!-- 52 --> [Tooltip] Fix broken arrow position in rtl (#27868) @mnajdova
- <!-- 02 --> [transitions] Allow to run Slide into a custom container (#26623) @benny0642

### `@material-ui/system@5.0.0-beta.5`

#### Breaking changes

- <!-- 40 --> [system] Rename `styleProps` to `ownerState` (#27830) @mnajdova

  The change was done in order to better reflect what they are, not what we think they will be used for.

  ```diff

>   <SomeSlotComponent
>   -  styleProps={propsAndState}
>   +  ownerState={propsAndState}
>   />

Reviewer 2: ```suggestion
   <SomeSlotComponent
  -  styleProps={propsAndState}
  +  ownerState={propsAndState}
   />
```

### [Versions](https://material-ui.com/versions/)

## 5.0.0-beta.5

<!-- generated comparing v5.0.0-beta.4..next -->

_Aug 24, 2021_

> Big thanks to the 26 contributors who made this release possible. Here are some highlights ✨:

Reviewer 4: ```suggestion
A big thanks to the 26 contributors who made this release possible. Here are some highlights ✨:
```

Author: We should change the template for `release:changelog` with this change :)

> - 🐛 Fixed lots of regressions and bugs to get us closer to the [v5 stable release milestone](https://github.com/mui-org/material-ui/milestone/44)

Reviewer 4: ```suggestion
- 🐛 Fixed a lot of bugs and regressions to get us closer to the [v5 stable release milestone](https://github.com/mui-org/material-ui/milestone/44)
```
I've no idea why "bugs and regressions" reads better, but it does 😅

- <!-- 15 --> [test] Add current behavior for a11y name vs visible name for PickersDay (#27661) @eps1lon
- <!-- 17 --> [test] Dodge double logging in dev mode (#27653) @oliviertassinari
- <!-- 14 --> [test] Enable skipped test fixed by upstream React fix (#27615) @eps1lon

> - <!-- 19 --> [theme] Add missed variants in Components interface (#27453) @nikitabobers

Reviewer: What's the reasoning here? We tagged this change for `beta.4`: https://github.com/mui-org/material-ui/commit/fc33c8170b0df49c68307d2c1c6cecfd5f878fe5. Was it not actually part of the beta.4 release?

Author: My bad, I must have moved the incorrect line. Thanks for pointing out will revert.
